build.gradle

```diff
@@ -56,30 +56,30 @@ shadowJar {
 }
 
 task prepareManifest(type: Copy) {
-    from 'launch4j/liteloader_installer_manifest.xml'
-    into tasks.launch4j.temporaryDir
+    from 'createExe/liteloader_installer_manifest.xml'
+    into tasks.createExe.temporaryDir
     expand(
         artefact:  project.sxsname,
         msversion: project.msversion
     )
 }
 
-launch4j {
+createExe {
```

Comment:
This should have `dependsOn 'prepareManifest'` (not sure if things have changed since the above comment, but `depensOn` works fine on my system)

Reply:
See line 19.

Reply:
Saw that. That comment isn't (is no longer?) correct. `defaultTasks` is used when running gradle without specifying a task, however if you simply run the `createExe` or `gather` task manually it will fail. Adding the `dependsOn` line fixes this.

Reply:
the dependson should be on a separate line at best.

Reply:
Finally got time to work on this! Added dependsOn (And on a separate line too).
Since it does indeed work, the comment was also removed